art/attacks/poisoning/adversarial_embedding_attack.py

```diff
@@ -0,0 +1,281 @@
+# MIT License
+#
+# Copyright (C) The Adversarial Robustness Toolbox (ART) Authors 2020
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""
+This module implements clean-label attacks on Neural Networks.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+from typing import Tuple, Union, List, TypeVar, Optional
+
+import numpy as np
+
+from art.attacks.attack import PoisoningAttackTransformer
+from art.attacks.poisoning import PoisoningAttackBackdoor
+from art.estimators import BaseEstimator, NeuralNetworkMixin
+from art.estimators.classification.classifier import ClassifierNeuralNetwork
+from art.estimators.classification.keras import KerasClassifier
+
+logger = logging.getLogger(__name__)
+
+
+class PoisoningAttackAdversarialEmbedding(PoisoningAttackTransformer):
+    """
+    Implementation of Adversarial Embedding attack by Tan, Shokri (2019).
+    "Bypassing Backdoor Detection Algorithms in Deep Learning"
+
+    This attack trains a classifier with an additional
+
+    | Paper link: https://arxiv.org/abs/1905.13409
+    """
+
+    attack_params = PoisoningAttackTransformer.attack_params + PoisoningAttackBackdoor.attack_params + [
```

Comment:
I think `PoisoningAttackBackdoor.attack_params` is not required here because `attack_params` should only contain the parameters in defined in `__init__.
```suggestion
    attack_params = PoisoningAttackTransformer.attack_params + [
```